CONTRIBUTING.md

```diff
@@ -1,86 +1,81 @@
+1. The bug fixer creates a pull request that fixes the issue.
+1. Others review the pull request.
+1. The bug fixer handles all code review comments.
+1. A gate-keeper accepts the pull request.
+1. The pull request merges successfully.
 
 ### Code review
-The code review process is designed to catch both style and domain specific issues. It is also designed to follow and respect the _definition of done_. Please make sure your code follows the style of the source code you are modifying. Each pull request must be reviewed by the gate-keeper before we can merge it to the master branch.
+
+The code review process catches both style and domain-specific issues. It also follows and respects the _definition of done_. Please make sure your code follows the style of the source code you are modifying. A gate-keeper must review each pull request before we can merge the pull request to the master branch.
+
 ## Issues and bug reporting
-Please report all bugs using the Issues tab on the GitHub web-page. It will help us to collaborate on issues more promptly. One of our gate-keepers (developers responsible for quality and the repository) will review the issue and assign a label such as _bug_, _enhancement_, _help wanted_ or _wontfix_.
-# How to contribute
-You can either file a bug, help fix a bug or propose a new feature (or enhancement) and implement it yourself. If you want to contribute please:
-* Bug reports: File a bug report in the Issues tab of this repo to let us know there are problems with our code.
+
+Please report all bugs using the Issues tab on the GitHub page. This process helps us collaborate on issues promptly. One of our gate-keepers (developers responsible for quality and the repository) will review the issue and assign a label, such as _bug_, _enhancement_, _help wanted_ or _wontfix_.
+
+## How to contribute
+
+You can file a bug, help fix a bug or propose a new feature (or enhancement) and implement it yourself. If you want to contribute, please:
+
+* Bug reports: File a bug report in the Issues tab of this repo to let us know there are problems with the code.
   * Make sure your bug report contains a simple description of the problem.
   * Include information about the host computer's configuration and OS or VM used.
-  * Include information about the application's version. All applications should have at least a ``--version`` switch you can use to check the version.
-  * Copy/paste useful console dumps and configuration files' content. Please use [fenced code blocks](https://help.github.com/articles/github-flavored-markdown/#fenced-code-blocks) to encapsulate code snippets.
+  * Include information about the application's version. All applications should have at least a `--version` switch you can use to check the version.
+  * Copy and paste useful console dumps and configuration files' content. Please use [fenced code blocks](https://help.github.com/articles/github-flavored-markdown/#fenced-code-blocks) to encapsulate code snippets.
```

Comment:
Looks like the link is dead, should be this one here: `https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code`